src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -104,11 +106,32 @@ public boolean delete(Long id) {
     private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException {
         Long id = resultSet.getObject("id", Long.class);
         String name = resultSet.getString("name");
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
         String licenseNumber = resultSet.getString("license_number");
         Driver driver = new Driver();
         driver.setId(id);
         driver.setName(name);
         driver.setLicenseNumber(licenseNumber);
+        driver.setPassword(password);
+        driver.setLogin(login);
         return driver;
     }
```

Comment:
Repeating code in CarDaoImpl and DriverDaoImpl. Tou can create separate service ParseDriverService, and move this methode to this service. When using yout new service, don`t forget about annotation